src/spec-node/containerFeatures.ts

```diff
@@ -82,7 +82,7 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 				args.push('--load'); // (short for --output=docker, i.e. load into normal 'docker images' collection)
 			}
 		}
-		if (params.buildxCacheTo) {
+		if (params.buildxCacheTo && !params.buildNoCache) {
```

Comment:
Is it possible to add tests to validate this change?

Reply:
`buildxCacheTo` can be controlled with a separate CLI option. I would not let `buildNoCache` affect its use here.